src/main/target/common_post.h

```diff
@@ -121,6 +121,10 @@
 
 #endif // defined(USE_RX_CC2500)
 
+#if defined(CAMERA_CONTROL_PIN) && defined(USE_OSD_SD) && !defined(USE_CAMERA_CONTROL)
```

Comment:
Curious as to why `&& defined(USE_OSD_SD)` is necessary?

Reply:
Added in last push as it would only be needed when using analog - so it should be USE_VTX instead?